media-proxy/src/api_server_tcp.cc

```diff
@@ -295,22 +298,33 @@ void RunTCPServer(ProxyContext* ctx)
     }
 
     INFO("TCP Server listening on %s", ctx->getTCPListenAddress().c_str());
-
     // signal(SIGINT, intHandler);
 
     while (keepRunning) {
         /* accept incoming connections */
-        connection = (connection_t*)malloc(sizeof(connection_t));
+        connection = (connection_t*)malloc(connection_t_size);
+        if (!connection) continue;
+
+        memset(connection, 0x0, connection_t_size);
```

Comment:
Oh, what's the reason to create the separate `connection_t_size` variable for storing the size?

Reply:
As this is being used inside a loop and in two separate places one after another so It is better to have this calculated.

Reply:
Don't optimize prematurely (c)
I suggest not to introduce a separate variable to store the size but keep it as it was because seeing `sizeof(...)` makes it easy to read the code.